elasticdl/python/tests/worker_test.py

```diff
@@ -160,53 +131,6 @@ def test_embedding_layer(self):
         )
         self.assertTrue(len(worker._embedding_layers) == 2)
 
-    def test_lookup_embedding(self):
```

Comment:
Formerly, this test case is for the correctness of `worker.lookup_embedding`. Maybe do not delete it but moving it to `layer_test.py`?

Reply:
It seems that former code compares the same thing:

```python
e_lookup, e_unknown = EmbeddingService.lookup_embedding(
   keys=["-".join([layer_name, str(id)]) for id in ids]
)
lookup_result = worker.lookup_embedding(
   ids=ids,
   layer_name=layer_name,
   embedding_table_dim=embedding_table_dim,
)
```

`worker.lookup_embedding` is a thin wrapper of `EmbeddingService.lookup_embedding`. It directly calls `EmbeddingService.lookup_embedding`. I think it's no need to make an unittest for these two functions.